test/development/app-dir/capture-console-error-owner-stack/capture-console-error-owner-stack.test.ts

```diff
@@ -58,8 +60,7 @@ describe('app-dir - capture-console-error-owner-stack', () => {
     } else {
       expect(result).toMatchInlineSnapshot(`
         {
-          "callStacks": "button
-        app/browser/event/page.js (5:6)",
```

Comment:
Why is this gone? `app/browser/event/page.js (5:6)` looks like product code to me so I'm confused why this would be ignore-listed.

Reply:
looks like timing issue... need to get it back